src/Views/Appearance.vala

```diff
@@ -110,6 +110,34 @@ public class PantheonShell.Appearance : Gtk.Grid {
         };
         dark_info.get_style_context ().add_class (Gtk.STYLE_CLASS_DIM_LABEL);
 
+        var schedule_label = new Gtk.Label (_("Schedule:")) {
+            halign = Gtk.Align.END,
+            xalign = 1
+        };
+
+        var schedule_mode_button = new Granite.Widgets.ModeButton ();
+        schedule_mode_button.append_text (_("Disabled"));
+        schedule_mode_button.append_text (_("Sunset to Sunrise"));
+        schedule_mode_button.append_text (_("Manual"));
+
+        var from_label = new Gtk.Label (_("From:"));
+
+        var from_time = new Granite.Widgets.TimePicker ();
+
+        var to_label = new Gtk.Label (_("To:"));
+
+        var to_time = new Granite.Widgets.TimePicker ();
+
+        var schedule_grid = new Gtk.Grid () {
+            column_spacing = 12,
+            row_spacing = 12
```

Comment:
Since there are no rows, row spacing doesn't do anything here :) But we can add some bottom margin to make sure we get double spacing between sections
```suggestion
            column_spacing = 12,
            margin_bottom = 24
```